### ISSUE TITLE
Unused files in models/masks/region1/man

### ISSUE THREAD

**Issue by zatinu322** [DeusExMachinaTeam/EM-CommunityPatch#476]
<details>
<summary>Хех</summary>

![image](https://github.com/DeusExMachinaTeam/EM-CommunityPatch/assets/68562524/4228b026-798e-45df-a893-c1d6ddce1725)

![image](https://github.com/DeusExMachinaTeam/EM-CommunityPatch/assets/68562524/746bbeb7-8378-48d3-ab1f-301ccaf0e442)
</details>

И ещё нужно с .sam моделькой разобраться

**Comment by Zvetkov** [comment]
Да, текстуры не лишние, а предназначены для новой модели забранной из м113, как моддерский ресурс. А вот то что новая модель не была сконвертирована в gam из sam - ошибка, необходимо исправить.
